data/World/Ganons Castle MQ.json

```diff
@@ -81,7 +94,11 @@
                     (has_explosives or Progressive_Strength_Upgrade or can_use(Dins_Fire)))",
             "Ganons Castle MQ Shadow Trial Eye Switch Chest": "
                 Bow and (logic_lens_castle_mq or can_use(Lens_of_Truth)) and 
-                (Hover_Boots or (Progressive_Hookshot and (has_fire_source or logic_shadow_trial_mq)))"
+                (Hover_Boots or (Progressive_Hookshot and (has_fire_source or logic_shadow_trial_mq)))",
+            "Ganons Castle MQ Shadow Trial Pot 1" : "(logic_lens_castle_mq or can_use(Lens_of_Truth)) and 
+            (Hover_Boots or (Progressive_Hookshot and (has_fire_source or logic_shadow_trial_mq)))",
+            "Ganons Castle MQ Shadow Trial Pot 2" : "(logic_lens_castle_mq or can_use(Lens_of_Truth)) and 
+            (Hover_Boots or (Progressive_Hookshot and (has_fire_source or logic_shadow_trial_mq)))"
```

Comment:
Requested Change: logic_shadow_trial_mq is a precise bow shot, so the trick route needs to also check for a bow.